packages/sdk/odahuflow/sdk/models/toolchain_integration_status.py

```diff
@@ -15,16 +15,27 @@ class ToolchainIntegrationStatus(Model):
     Do not edit the class manually.
```

Comment:
I'm a little curious. How do the new fields look like in JSON format?

Reply:
"status": {
      "createdAt": "2020-03-04T22:04:44Z"
    }

Reply:
It seems that ISO dateformat is definitely OK for rest api